debian/patches/0003-tools-mpris-proxy-allow-user-to-specify-different-mp.patch

```diff
@@ -1,65 +0,0 @@
-From 7abc94204c2dd72fd6a731a0240123e344dd2ac9 Mon Sep 17 00:00:00 2001
-From: Simon Fels <simon.fels@canonical.com>
-Date: Mon, 19 Oct 2015 22:13:02 +0200
-Subject: [PATCH 3/4] tools: mpris-proxy: allow user to specify different mpris
```

Comment:
Is this not needed? is this a ubuntu touch thing or ubuntu desktop thing?

Reply:
I have no idea what that is ^^

Reply:
While I still have no idea this thing was not even in series, so deactivated already... SO I guess its fine to be removed.